WooCommerce/Classes/ViewRelated/Themes/ThemesCarouselViewModel.swift

```diff
@@ -41,6 +45,10 @@ final class ThemesCarouselViewModel: ObservableObject {
     func updateCurrentTheme(id: String?) {
         currentThemeID = id
     }
+
+    func updateReloadBehavior(action: @escaping () -> Void) {
```

Comment:
Opinion: If this is only about setting the `onReload` callback, consider renaming it into something like `setDidReloadCallback` for clarity.

Reply:
I cannot find a better name for this so updated `onReload` to be mutable for simplicity in 430d8bd622298cd6f941b526132f183e6395836d.